README.md

```diff
@@ -157,6 +163,63 @@ describe('Pact', () => {
   })
 })
 ```
+
+#### Provider API Testing
+
+Once you have created Pacts for your Consumer, you need to validate those Pacts against your Provider.
+
+First, install [Pact Node](https://github.com/pact-foundation/pact-node):
+
+```
+npm install @pact-foundation/pact-node --save
+```
+
+Then run the Provider side verification step:
+
+```js
+var pact = require('@pact-foundation/pact-node');
+var opts = {
+	providerBaseUrl: <String>,       // Running API provider host endpoint. Required.
+	pactUrls: <Array>,               // Array of local Pact file paths or Pact Broker URLs (http based). Required.
+	providerStatesUrl: <String>,     // URL to fetch the provider states for the given provider API. Optional.
+	providerStatesSetupUrl <String>, // URL to send PUT requests to setup a given provider state. Optional.
+	pactBrokerUsername: <String>,    // Username for Pact Broker basic authentication. Optional
+	pactBrokerPassword: <String>,    // Password for Pact Broker basic authentication. Optional
+};
+
+pact.verifyPacts(opts)).then(function () {
+	// do something
+});
+```
+
+That's it! Read more about [Verifying Pacts](http://docs.pact.io/documentation/verifying_pacts.html).
+
+### Publishing Pacts to a Broker
+
+Sharing is caring - to simplify sharing Pacts between Consumers and Providers, checkout [sharing pacts](http://docs.pact.io/documentation/sharings_pacts.html).
+
+```js
+var pact = require('@pact-foundation/pact-node');
+var opts = {
+	pactUrls: <Array>,               // Array of local Pact files or directories containing them. Required.
+	pactBroker: <String>,            // URL to fetch the provider states for the given provider API. Optional.
+	pactBrokerUsername: <String>,    // Username for Pact Broker basic authentication. Optional
+	pactBrokerPassword: <String>     // Password for Pact Broker basic authentication. Optional
+};
+
+pact.publishPacts(opts)).then(function () {
+	// do something
+});
+```
+
+#### Using Mocha?
+
+Check out [Pact JS Mocha](https://github.com/pact-foundation/pact-js-mocha).
+
+#### Using Jasmine?
```

Comment:
Fair enough, I noticed it had no docs so makes sense. When in doubt, leave it out!